src/extensions/averagewaveforms2/AverageWaveformsView/setupElectrodes.ts

```diff
@@ -0,0 +1,104 @@
+import { norm } from 'mathjs';
```

Comment:
Separated out the computation code required to determine the placement of electrodes on the canvas